app/views/pages/crayons.html.erb

```diff
@@ -50,6 +50,67 @@
   }
 </style>
 
+<div class="container">
```

Comment:
@nickytonline 
This is where I put frontend code for the elements that you will likely preactify :)